pkgs/development/python-modules/buildout/default.nix

```diff
@@ -1,18 +1,43 @@
-{ lib, buildPythonPackage, fetchPypi }:
+{ lib, buildPythonPackage, fetchFromGitHub
+, setuptools, pip, wheel
+, zope_testing, manuel/*, bobo*/, zdaemon
+}:
 
 buildPythonPackage rec {
-  pname = "zc.buildout";
-  version = "2.13.4";
+  pname = "zc_buildout";
+  version = "3.0.0b2";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "b978b2f9317b317ee4191f78fcc4f05b1ac41bdaaae47f0956f14c8285feef63";
+  src = fetchFromGitHub {
+    owner = "buildout";
+    repo = "buildout";
+    rev = version;
+    sha256 = "01sj09xx5kmkzynhq1xd8ahn6xqybfi8lrqjqr5lr45aaxjk2pid";
   };
 
+  propagatedBuildInputs = [
+    setuptools
+    pip
+    wheel
+  ];
+
+  checkInputs = [
+    zope_testing
+    manuel
+    # bobo # Missing package
+    zdaemon
+    # zc_zdaemonrecipe # Missing package
+    # zc_recipe_deployment # Missing package
```

Comment:
Same here. Does the default build/check fail or produce errors without these packages?

Reply:
They are required, as you can check at: https://github.com/buildout/buildout/blob/master/setup.py#L57

Reply:
I haven't fixed tests. I suppose the lack of dependencies will block tests. But I'm not familiar with this test suite. I'm only trying to bump version to fix another package that got out of sync with another bumped dependency (setuptools).

Reply:
These dependencies are listed as required for tests in `setup.py`. https://github.com/buildout/buildout/blob/master/setup.py#L58